src/index.js

```diff
@@ -1 +1,5 @@
-import './style.css'
```

Comment:
- [ ] I noticed that you have implemented the whole todo list on this milestone. You are required to only implemented the UI and populate the todo tasks dynamically, user interactions will be added in upcoming milestones. I kindly suggest that you remove all code for future milestones and stash it somewhere. Create an array of tasks as demanded by the project requirements of this milestone and use it to populate your page by looping through it.